tests/api.c

```diff
@@ -15572,6 +15572,10 @@ static int test_wc_AesCbcEncryptDecrypt (void)
 #endif
     }
     if (cbcE == 0) {
+    #if defined(HAVE_FIPS) && defined(HAVE_FIPS_VERSION) && \
+        (HAVE_FIPS_VERSION == 2) && defined(WOLFSSL_AESNI)
+        printf("Zero length inputs not supported in FIPS mode (v2), skip test");
```

Comment:
Might include AESNI in the note.... Also should this be for v1 and v2 even though technically v1 doesn't support AESNI?

Reply:
will add AESNI to the note, no this shouldn't be v1, users can't enable AESNI with v1 as you noted.

Reply:
Updated note. The reason for specifically calling out (v2) is that this should work fine with 140-3 (v5) where we have added the check for zero length and return immediately upon entry when length is zero.